Library/Homebrew/cli/parser.rb

```diff
@@ -19,9 +20,18 @@ class Parser
 
       def self.from_cmd_path(cmd_path)
         cmd_args_method_name = Commands.args_method_name(cmd_path)
+        cmd_name = cmd_args_method_name.to_s.delete_suffix("_args").tr("_", "-")
 
         begin
-          Homebrew.send(cmd_args_method_name) if require?(cmd_path)
+          if require?(cmd_path)
+            cmd = Homebrew::AbstractCommand.command(cmd_name)
```

Comment:
Can be done in another PR: I think it would make sense to move `AbstractCommand` to `Homebrew::CLI::AbstractCommand`.

Reply:
I have no strong feelings on their either way.